.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -1779,12 +1694,7 @@ stages:
       parameters:
         includeX86: true
     - template: steps/restore-working-directory.yml
-    - task: DownloadPipelineArtifact@2
-      displayName: Download samples
-      inputs:
-        artifact: samples
-        path: $(outputDir)
-      retryCountOnTaskFailure: 5
+    - template: steps/download-samples.yml
```

Comment:
we don't currently run multi-version tests on windows, so don't need to specify the framework here